tests/ui/doc/doc_lazy_list.fixed

```diff
@@ -52,26 +52,32 @@ fn seven() {}
 ///
 /// # Arguments
 /// * `protocol_descriptors`: A Json Representation of the ProtocolDescriptors
-///     to set up. Example:
+///   to set up. Example:
 ///   'protocol_descriptors': [
 //~^ ERROR: doc list item without indentation
-///      {
-///          'protocol': 25,  # u64 Representation of ProtocolIdentifier::AVDTP
-///          'params': [
-///              {
-///                 'data': 0x0103  # to indicate 1.3
-///              },
-///              {
-///                  'data': 0x0105  # to indicate 1.5
-///              }
+///   {
+///   'protocol': 25,  # u64 Representation of ProtocolIdentifier::AVDTP
+///   'params': [
+///   {
+///   'data': 0x0103  # to indicate 1.3
+///   },
+///   {
+///   'data': 0x0105  # to indicate 1.5
+///   }
```

Comment:
It would be good to add [error annotations](https://rustc-dev-guide.rust-lang.org/tests/ui.html#error-annotations) here for consistency, like has already been done with it having no indentation.